source/docs/appendices/migration-guides.md

```diff
@@ -9,6 +9,12 @@ Our goal is to make migrations as smooth as possible. This is why we try to make
 
 ## `2.6.0` -> `2.7.0`
 
+### Additional headers in Magento API calls
+
+Both Magento 1 and Magento 2 modules in Front-Commerce now supports sending additional headers in all API calls. To do so, you must define the `magento.api.extraHeaders` (for storefront API) and/or `magento.api.extraAdminHeaders` (for admin API) configuration values from a [configuration provider](/docs/advanced/server/configurations.html#What-is-a-configuration-provider).
```

Comment:
IMHO, that does not belong to the migration guide as there's nothing to migrate
on that topic. I would rather put that under Magento1 and Magento2 section maybe
in "Advanced" page ?

Reply:
I agree. We could start with an "Advanced" page with miscellaneous and split it later if needed.

I created them instead of the migration guide:
- https://deploy-preview-185--heuristic-almeida-1a1f35.netlify.app/docs/magento1/advanced.html
- https://deploy-preview-185--heuristic-almeida-1a1f35.netlify.app/docs/magento2/advanced.html